src/features/ConnectWallet/model/useConnectWallet.ts

```diff
@@ -35,5 +41,19 @@ export const useConnectWallet = ({ isForm, onWalletConnect }: UseConnectWalletPr
     }
   }, [isConnected, hasClickedConnect]);
 
+  React.useEffect(() => {
+    if (isConnected && !address) {
+      triggerToast({
+        message: `Something went wrong`,
+        description:
+          'We were unable to complete the current operation. Try again or connect support.',
+        type: ToastType.Error,
+      });
+
+      Sentry.captureMessage('Wallet is connected, but no address found');
+      disconnect().catch(() => {});
```

Comment:
```suggestion
      disconnect();
```
Так что-то изменится?